Sources/BraintreeAmericanExpress/BTAmericanExpressAnalytics.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
-class BTAmericanExpressAnalytics {
-    
+enum BTAmericanExpressAnalytics {
+
     // MARK: - Get Rewards Balance Events
     
     static let started = "amex:rewards-balance:started"
```

Comment:
Is there a preference for `static let` vs `case` since these are enums now?

Reply:
Yeah the preference for constants is a static let in an enum, if we used case at the callsite it gets a bit more verbose as well and a bit more unclear that we are using a constant. Here is an example of how we'd call the caseless enum with a constant vs a standard enum:
```swift
enum MyEnum {
    static let name = "Name"
}

enum MyEnumTwo: String {
    case name = "Name"
}

print(MyEnum.name)
print(MyEnumTwo.name.rawValue)
```